app/src/main/java/lt/vilnius/tvarkau/entity/Problem.kt

```diff
@@ -0,0 +1,52 @@
+package lt.vilnius.tvarkau.entity
+
+import com.google.android.gms.maps.model.LatLng
+import com.google.gson.annotations.SerializedName
+import org.parceler.Parcel
+import org.threeten.bp.LocalDateTime
+
+@Parcel
+data class Problem(
```

Comment:
DTO pattern is good, but often missused by creating layers and layers of duplicate data objects.
IMO DTO are good when it's used with databases and filled selectively with only required data.
I don't think this would be a good use for it. Yes, we have extra fields, but now (with this) implementation it is hidden from end user.